src/explore-education-statistics-frontend/src/components/AccordionSection.tsx

```diff
@@ -45,17 +45,16 @@ const AccordionSection = ({
         'govuk-accordion__section--expanded': open,
       })}
     >
-      <div className="govuk-accordion__section-header">
+      <div
+        className="govuk-accordion__section-header"
+        data-testid={`SectionHeader ${heading}`}
```

Comment:
How come we're breaking convention with other test IDs with this format?

Reply:
The "heading" variable is normal text, not a slug, so I formatted it to match normal text. To be honest, this was the best I could come up with, given the situation.

Any suggestions on how I can do it better are welcome.